{
  "Faults": [
    {
      "py/object": "Fault",
      "loggedAt": "2022-11-01T01:18:00.000+0000",
      "latitude": 50.75053496973706,
      "estimatedRestoration": "2022-11-01T21:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KC8836'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KC8836",
      "reference": "KC8836",
      "maxLongitude": -1.6492,
      "engineerOnSiteTime": "2022-11-01T04:00:00.000+0000",
      "minLatitude": 50.7467,
      "name": "BH25 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "BH25 6BL",
        "BH25 6BY",
        "BH25 6BZ",
        "BH25 6EZ",
        "BH25 6HD",
        "BH25 6HE",
        "BH25 6JJ",
        "BH25 6LP",
        "BH25 6NN",
        "BH25 6NP",
        "BH25 6NT",
        "BH25 6NU",
        "BH25 6NW",
        "BH25 6PQ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.6596,
              50.7535
            ],
            [
              -1.6492,
              50.7535
            ],
            [
              -1.6492,
              50.7467
            ],
            [
              -1.6596,
              50.7467
            ],
            [
              -1.6596,
              50.7535
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-11-01T03:30:00.000+0000",
      "UUID": "1660595a-acab-49c1-9402-42d7278d4e70",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -1.6596,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 50.7535,
      "longitude": -1.653612549923052,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-11-01T00:14:00.000+0000",
      "latitude": 50.96401011875629,
      "estimatedRestoration": "2022-11-02T05:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KC8794'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KC8794",
      "reference": "KC8794",
      "maxLongitude": -2.2284,
      "engineerOnSiteTime": "2022-11-01T02:15:00.000+0000",
      "minLatitude": 50.9609,
      "name": "SP7 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SP7 0AT",
        "SP7 0LA",
        "SP7 0LB",
        "SP7 0LF"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.2398,
              50.9687
            ],
            [
              -2.2284,
              50.9687
            ],
            [
              -2.2284,
              50.9609
            ],
            [
              -2.2398,
              50.9609
            ],
            [
              -2.2398,
              50.9687
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "1fbca75e-ce06-4eaa-af7d-a7cf2545ce46",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -2.2398,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 50.9687,
      "longitude": -2.232840560202054,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-11-01T04:53:00.000+0000",
      "latitude": 50.78799572083963,
      "estimatedRestoration": "2022-11-01T23:59:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KC8955'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KC8955",
      "reference": "KC8955",
      "name": "BH23 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "BH23 7BG",
        "BH23 7BL",
        "BH23 7BQ"
      ],
      "location": {
        "coordinates": [
          -1.7928152726416708,
          50.78799572083963
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-11-01T18:30:00.000+0000",
      "UUID": "d38c33e9-6219-4848-8cfb-eaf71e54be35",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.7928152726416708,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-11-01T05:55:00.000+0000",
      "latitude": 50.85068695107029,
      "estimatedRestoration": "2022-11-01T23:59:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KC8981'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KC8981",
      "reference": "KC8981",
      "maxLongitude": -1.4059,
      "engineerOnSiteTime": "2022-11-01T08:00:00.000+0000",
      "minLatitude": 50.8457,
      "name": "SO45 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SO45 4HN",
        "SO45 4HR",
        "SO45 4HS",
        "SO45 4JZ",
        "SO45 4LA",
        "SO45 4LB",
        "SO45 4LF",
        "SO45 4QH",
        "SO45 4QJ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.4163,
              50.8525
            ],
            [
              -1.4059,
              50.8525
            ],
            [
              -1.4059,
              50.8457
            ],
            [
              -1.4163,
              50.8457
            ],
            [
              -1.4163,
              50.8525
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-11-01T08:00:00.000+0000",
      "UUID": "993bc5e1-a488-4a6b-9ff4-3db374acf58a",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -1.4163,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 50.8525,
      "longitude": -1.4097193585285148,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-11-01T06:45:00.000+0000",
      "latitude": 50.785124650346155,
      "estimatedRestoration": "2022-11-01T18:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KC9019'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KC9019",
      "reference": "KC9019",
      "maxLongitude": -1.3426,
      "minLatitude": 50.7828,
      "name": "SO45 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SO45 1AD"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.3741,
              50.7917
            ],
            [
              -1.3426,
              50.7917
            ],
            [
              -1.3426,
              50.7828
            ],
            [
              -1.3741,
              50.7828
            ],
            [
              -1.3741,
              50.7917
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-11-01T14:00:00.000+0000",
      "UUID": "1ae5c19e-6fc2-4fd7-a3a2-34c1d6424935",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -1.3741,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "evChargersAffected": [
        "cd0d5d8a-cd77-422f-a5fa-c2f3fc6a6dd0"
      ],
      "maxLatitude": 50.7917,
      "longitude": -1.3624134644062649,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-11-01T05:19:00.000+0000",
      "latitude": 50.62156173577667,
      "estimatedRestoration": "2022-11-01T21:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KC8960'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KC8960",
      "reference": "KC8960",
      "maxLongitude": -1.9877,
      "engineerOnSiteTime": "2022-11-01T07:10:00.000+0000",
      "minLatitude": 50.6089,
      "name": "BH19 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "BH19 3BD",
        "BH19 3DS",
        "BH19 3DT",
        "BH19 3DX",
        "BH19 3DY",
        "BH19 3DZ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.0234,
              50.6263
            ],
            [
              -1.9877,
              50.6263
            ],
            [
              -1.9877,
              50.6089
            ],
            [
              -2.0234,
              50.6089
            ],
            [
              -2.0234,
              50.6263
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "59044d76-493a-4362-829a-38427a712098",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -2.0234,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 50.6263,
      "longitude": -2.0165716612803015,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-11-01T07:42:00.000+0000",
      "latitude": 51.55979164809482,
      "estimatedRestoration": "2022-11-01T23:59:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KC9090'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KC9090",
      "reference": "KC9090",
      "maxLongitude": -1.7548,
      "engineerOnSiteTime": "2022-11-01T08:30:00.000+0000",
      "minLatitude": 51.5579,
      "name": "SN3 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SN3 3AF",
        "SN3 3AH",
        "SN3 3AJ",
        "SN3 3AL",
        "SN3 3BS"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.7613,
              51.5611
            ],
            [
              -1.7548,
              51.5611
            ],
            [
              -1.7548,
              51.5579
            ],
            [
              -1.7613,
              51.5579
            ],
            [
              -1.7613,
              51.5611
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-11-01T09:30:00.000+0000",
      "UUID": "314024f1-445e-4327-ac13-05aba74e560e",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -1.7613,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 51.5611,
      "longitude": -1.7579717676780373,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-11-01T07:29:00.000+0000",
      "latitude": 50.84363135413397,
      "estimatedRestoration": "2022-11-01T19:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KC9069'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KC9069",
      "reference": "KC9069",
      "engineerOnSiteTime": "2022-11-01T09:00:00.000+0000",
      "name": "BH21 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "BH21 6FA",
        "BH21 6FB",
        "BH21 6RE",
        "BH21 6SP",
        "BH21 6SX",
        "BH21 6SY"
      ],
      "location": {
        "coordinates": [
          -1.865828872686984,
          50.84363135413397
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-11-01T08:45:00.000+0000",
      "UUID": "61706f00-ef78-40fb-a08d-aec2700c2c33",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.865828872686984,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-11-01T07:10:00.000+0000",
      "latitude": 50.994447612904736,
      "estimatedRestoration": "2022-11-01T22:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KC9049'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KC9049",
      "reference": "KC9049",
      "maxLongitude": -1.3119,
      "minLatitude": 50.9916,
      "name": "SO21 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SO21 1AJ",
        "SO21 1DL",
        "SO21 1GN",
        "SO21 1TQ",
        "SO21 1XB"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.3177,
              50.9971
            ],
            [
              -1.3119,
              50.9971
            ],
            [
              -1.3119,
              50.9916
            ],
            [
              -1.3177,
              50.9916
            ],
            [
              -1.3177,
              50.9971
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-11-01T16:30:00.000+0000",
      "UUID": "da7e3662-538b-4e74-b806-27a5b841eb90",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -1.3177,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 50.9971,
      "longitude": -1.3154491131053432,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-11-01T08:34:00.000+0000",
      "latitude": 50.763016770462876,
      "estimatedRestoration": "2022-11-01T20:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KC9168'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KC9168",
      "reference": "KC9168",
      "maxLongitude": -1.602,
      "minLatitude": 50.7597,
      "name": "SO41 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SO41 0FL",
        "SO41 0FN",
        "SO41 6DF"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.6221,
              50.7679
            ],
            [
              -1.602,
              50.7679
            ],
            [
              -1.602,
              50.7597
            ],
            [
              -1.6221,
              50.7597
            ],
            [
              -1.6221,
              50.7679
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "a729b89f-59a2-486f-b076-3128e737868c",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -1.6221,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 50.7679,
      "longitude": -1.6108283357210584,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-11-01T07:06:00.000+0000",
      "latitude": 51.277046614449254,
      "estimatedRestoration": "2022-11-01T19:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KC9039'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KC9039",
      "reference": "KC9039",
      "maxLongitude": -2.1612,
      "engineerOnSiteTime": "2022-11-01T08:05:00.000+0000",
      "minLatitude": 51.2734,
      "name": "BA13 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "BA13 4NB"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.1842,
              51.2861
            ],
            [
              -2.1612,
              51.2861
            ],
            [
              -2.1612,
              51.2734
            ],
            [
              -2.1842,
              51.2734
            ],
            [
              -2.1842,
              51.2861
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "b9342475-cc37-4e36-b99d-36f646f18c59",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -2.1842,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 51.2861,
      "longitude": -2.1730903909668293,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-10-31T14:50:00.000+0000",
      "latitude": 50.67844188606591,
      "estimatedRestoration": "2022-11-01T18:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KC8758'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KC8758",
      "reference": "KC8758",
      "maxLongitude": -2.0728,
      "minLatitude": 50.6771,
      "name": "BH20 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "BH20 5BG"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.0971,
              50.6884
            ],
            [
              -2.0728,
              50.6884
            ],
            [
              -2.0728,
              50.6771
            ],
            [
              -2.0971,
              50.6771
            ],
            [
              -2.0971,
              50.6884
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "6c3797c9-bb19-457d-970d-55df088fd68c",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -2.0971,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 50.6884,
      "longitude": -2.091971793776291,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-11-01T08:53:00.000+0000",
      "latitude": 51.473715056170086,
      "estimatedRestoration": "2022-11-01T21:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KC9205'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KC9205",
      "reference": "KC9205",
      "maxLongitude": -0.3945,
      "engineerOnSiteTime": "2022-11-01T11:00:00.000+0000",
      "minLatitude": 51.4706,
      "name": "TW4 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "TW4 7NE",
        "TW4 7NF",
        "TW4 7NG",
        "TW4 7NJ",
        "TW4 7NR"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.4033,
              51.4754
            ],
            [
              -0.3945,
              51.4754
            ],
            [
              -0.3945,
              51.4706
            ],
            [
              -0.4033,
              51.4706
            ],
            [
              -0.4033,
              51.4754
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-11-01T10:45:00.000+0000",
      "UUID": "f814f1d6-74dc-45a4-901f-2044040c38d8",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -0.4033,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 51.4754,
      "longitude": -0.3993077108944054,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-11-01T08:00:00.000+0000",
      "latitude": 50.63975602186083,
      "estimatedRestoration": "2022-11-01T20:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KC9400'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KC9400",
      "reference": "KC9400",
      "maxLongitude": -1.285,
      "minLatitude": 50.6257,
      "name": "PO30 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "PO30 3EL",
        "PO30 3EU",
        "PO30 3HB",
        "PO30 3HD",
        "PO30 3HE",
        "PO30 3HF",
        "PO30 3HG",
        "PO30 3HH",
        "PO30 3HQ",
        "PO38 2JY",
        "PO38 2JZ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.3367,
              50.6555
            ],
            [
              -1.285,
              50.6555
            ],
            [
              -1.285,
              50.6257
            ],
            [
              -1.3367,
              50.6257
            ],
            [
              -1.3367,
              50.6555
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "014e67f2-0420-4d1c-8298-784483cb9d4f",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -1.3367,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 50.6555,
      "longitude": -1.3102753409187822,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-11-01T09:23:00.000+0000",
      "latitude": 50.74819661689535,
      "estimatedRestoration": "2022-11-01T21:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KC9247'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KC9247",
      "reference": "KC9247",
      "maxLongitude": -1.849,
      "engineerOnSiteTime": "2022-11-01T11:02:00.000+0000",
      "minLatitude": 50.7438,
      "name": "BH8 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "BH8 9HX",
        "BH8 9PJ",
        "BH8 9PS",
        "BH8 9PT",
        "BH8 9PZ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.8592,
              50.751
            ],
            [
              -1.849,
              50.751
            ],
            [
              -1.849,
              50.7438
            ],
            [
              -1.8592,
              50.7438
            ],
            [
              -1.8592,
              50.751
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "ddbc3dda-1ba2-4e8b-8643-8e1e6c8f1aa9",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -1.8592,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 50.751,
      "longitude": -1.8535996497597185,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-11-01T11:24:00.000+0000",
      "latitude": 50.954458795449305,
      "estimatedRestoration": "2022-11-01T23:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KC9446'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KC9446",
      "reference": "KC9446",
      "maxLongitude": -1.2223,
      "engineerOnSiteTime": "2022-11-01T12:10:00.000+0000",
      "minLatitude": 50.9509,
      "name": "SO32 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SO32 1JU",
        "SO32 1RL",
        "SO32 1SQ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.2265,
              50.9556
            ],
            [
              -1.2223,
              50.9556
            ],
            [
              -1.2223,
              50.9509
            ],
            [
              -1.2265,
              50.9509
            ],
            [
              -1.2265,
              50.9556
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-11-01T12:15:00.000+0000",
      "UUID": "cb57f02e-6b07-472d-80b6-4e6313f84c2b",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -1.2265,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 50.9556,
      "longitude": -1.2234955540042793,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-11-01T12:13:00.000+0000",
      "latitude": 51.177862770949666,
      "estimatedRestoration": "2022-11-01T22:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KC9514'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KC9514",
      "reference": "KC9514",
      "maxLongitude": -2.2533,
      "engineerOnSiteTime": "2022-11-01T13:10:00.000+0000",
      "minLatitude": 51.0896,
      "name": "BA12 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "BA12 6JB",
        "BA12 7PP",
        "BA12 7PR",
        "BA12 7QL"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.2757,
              51.2082
            ],
            [
              -2.2533,
              51.2082
            ],
            [
              -2.2533,
              51.0896
            ],
            [
              -2.2757,
              51.0896
            ],
            [
              -2.2757,
              51.2082
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-11-01T13:10:00.000+0000",
      "UUID": "11bafc65-cd5d-448d-9a7d-dd2342f1fec0",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -2.2757,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 51.2082,
      "longitude": -2.26188470616763,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-11-01T12:04:00.000+0000",
      "latitude": 51.53548949559112,
      "estimatedRestoration": "2022-11-01T23:59:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KC9494'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KC9494",
      "reference": "KC9494",
      "maxLongitude": -0.7816,
      "engineerOnSiteTime": "2022-11-01T12:45:00.000+0000",
      "minLatitude": 51.5303,
      "name": "SL6 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SL6 5LH"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.8062,
              51.5447
            ],
            [
              -0.7816,
              51.5447
            ],
            [
              -0.7816,
              51.5303
            ],
            [
              -0.8062,
              51.5303
            ],
            [
              -0.8062,
              51.5447
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "f3e59cdd-5624-4cb1-bd6f-d0d10bf5f6d6",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -0.8062,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 51.5447,
      "longitude": -0.7911084156216117,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-11-01T13:20:00.000+0000",
      "latitude": 51.351112798814874,
      "estimatedRestoration": "2022-11-01T22:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KC9511'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KC9511",
      "reference": "KC9511",
      "engineerOnSiteTime": "2022-11-01T13:00:00.000+0000",
      "name": "SN8 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SN8 3SR"
      ],
      "location": {
        "coordinates": [
          -1.615577454228902,
          51.351112798814874
        ],
        "type": "Point"
      },
      "UUID": "3c713bdf-6c75-4281-b51a-bb2686049e19",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.615577454228902,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-11-01T13:46:00.000+0000",
      "latitude": 51.74866156991681,
      "estimatedRestoration": "2022-11-01T21:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KC9645'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KC9645",
      "reference": "KC9645",
      "maxLongitude": -1.2671,
      "engineerOnSiteTime": "2022-11-01T14:45:00.000+0000",
      "minLatitude": 51.747,
      "name": "OX2 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "OX2 0NX",
        "OX2 0NY"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.2708,
              51.7505
            ],
            [
              -1.2671,
              51.7505
            ],
            [
              -1.2671,
              51.747
            ],
            [
              -1.2708,
              51.747
            ],
            [
              -1.2708,
              51.7505
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "ad76545c-f181-4740-82ea-113afc88f94f",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -1.2708,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 51.7505,
      "longitude": -1.269072601322345,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-11-01T16:49:00.000+0000",
      "latitude": 50.87233712362124,
      "estimatedRestoration": "2022-11-01T20:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KC9851'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KC9851",
      "reference": "KC9851",
      "maxLongitude": -1.0391,
      "minLatitude": 50.8697,
      "name": "PO7 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "PO7 5JU",
        "PO7 5SS",
        "PO7 5ST",
        "PO7 5SY",
        "PO7 5TB",
        "PO7 5TF",
        "PO7 5TS"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.0441,
              50.8752
            ],
            [
              -1.0391,
              50.8752
            ],
            [
              -1.0391,
              50.8697
            ],
            [
              -1.0441,
              50.8697
            ],
            [
              -1.0441,
              50.8752
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-11-01T17:00:00.000+0000",
      "UUID": "d121bc8f-37ba-447b-9d06-ec367a628c79",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -1.0441,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 50.8752,
      "longitude": -1.0417228498783178,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-11-01T17:09:00.000+0000",
      "latitude": 50.77290447146923,
      "estimatedRestoration": "2022-11-01T20:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KC9865'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KC9865",
      "reference": "KC9865",
      "maxLongitude": -0.8097,
      "minLatitude": 50.7604,
      "name": "PO20 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "PO20 7JR",
        "PO20 7JS",
        "PO20 7JT",
        "PO20 7JU",
        "PO20 7JX",
        "PO20 7JY",
        "PO20 7JZ",
        "PO20 7LA",
        "PO20 7LB",
        "PO20 7LD",
        "PO20 7LE",
        "PO20 7LF",
        "PO20 7NU",
        "PO20 7QJ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.8456,
              50.7888
            ],
            [
              -0.8097,
              50.7888
            ],
            [
              -0.8097,
              50.7604
            ],
            [
              -0.8456,
              50.7604
            ],
            [
              -0.8456,
              50.7888
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "7139c0b6-a7b7-4c17-b9af-79b5ea805365",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -0.8456,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 50.7888,
      "longitude": -0.8289527948785508,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-11-01T16:11:00.000+0000",
      "latitude": 51.404804699462126,
      "estimatedRestoration": "2022-11-01T20:10:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KC9817'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KC9817",
      "reference": "KC9817",
      "maxLongitude": -1.3016,
      "minLatitude": 51.4019,
      "name": "RG14 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "RG14 2AX",
        "RG14 2BA",
        "RG14 2BB",
        "RG14 2BN",
        "RG14 2BP",
        "RG14 2BT",
        "RG14 2BX",
        "RG14 2DH",
        "RG14 2ER"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.3167,
              51.407
            ],
            [
              -1.3016,
              51.407
            ],
            [
              -1.3016,
              51.4019
            ],
            [
              -1.3167,
              51.4019
            ],
            [
              -1.3167,
              51.407
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-11-01T18:00:00.000+0000",
      "UUID": "24f5a64c-506e-45ee-a714-7aff2f4c886b",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -1.3167,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 51.407,
      "longitude": -1.310032452617816,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-11-01T17:18:00.000+0000",
      "latitude": 51.099650255512906,
      "estimatedRestoration": "2022-11-01T20:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KC9874'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KC9874",
      "reference": "KC9874",
      "name": "SP4 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SP4 6DZ",
        "SP4 6EA",
        "SP4 6QX"
      ],
      "location": {
        "coordinates": [
          -1.7867857673227694,
          51.099650255512906
        ],
        "type": "Point"
      },
      "UUID": "d23ecb0d-4d60-4e60-b9a0-49d4b52d46d3",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.7867857673227694,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-11-01T17:53:00.000+0000",
      "latitude": 50.88682336802539,
      "estimatedRestoration": "2022-11-01T20:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KC9882'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KC9882",
      "reference": "KC9882",
      "maxLongitude": -1.0703,
      "minLatitude": 50.8726,
      "name": "PO7 & PO17 Areas",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "PO17 6ER",
        "PO17 6ES",
        "PO17 6ET",
        "PO17 6EU",
        "PO7 6UB",
        "PO7 6UD"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.1019,
              50.9044
            ],
            [
              -1.0703,
              50.9044
            ],
            [
              -1.0703,
              50.8726
            ],
            [
              -1.1019,
              50.8726
            ],
            [
              -1.1019,
              50.9044
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-11-01T19:00:00.000+0000",
      "UUID": "484e57a5-1b01-4d01-9803-a8e8fdade9c2",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -1.1019,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 50.9044,
      "longitude": -1.0818861901324583,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-11-01T17:28:00.000+0000",
      "latitude": 51.51029464322547,
      "estimatedRestoration": "2022-11-01T21:50:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KC9922'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KC9922",
      "reference": "KC9922",
      "maxLongitude": -0.5362,
      "minLatitude": 51.5049,
      "name": "SL3 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SL3 6BD",
        "SL3 6DB",
        "SL3 6DD",
        "SL3 6ED",
        "SL3 6EG",
        "SL3 6EZ",
        "SL3 8BY"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.561,
              51.5223
            ],
            [
              -0.5362,
              51.5223
            ],
            [
              -0.5362,
              51.5049
            ],
            [
              -0.561,
              51.5049
            ],
            [
              -0.561,
              51.5223
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-11-01T19:30:00.000+0000",
      "UUID": "ba4600ce-81ac-4054-a257-0fdf49dc759c",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -0.561,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 51.5223,
      "longitude": -0.5448839994210389,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-11-01T07:00:00.000+0000",
      "latitude": 50.785124650346155,
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. We are working hard to get the power back on as quickly as we can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KC9945'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KC9945",
      "reference": "KC9945",
      "maxLongitude": -1.3426,
      "minLatitude": 50.7828,
      "name": "SO45 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SO45 1AD"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.3741,
              50.7917
            ],
            [
              -1.3426,
              50.7917
            ],
            [
              -1.3426,
              50.7828
            ],
            [
              -1.3741,
              50.7828
            ],
            [
              -1.3741,
              50.7917
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "3ce262f5-75f0-4286-8ce0-c28f4aaef99e",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -1.3741,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "evChargersAffected": [
        "cd0d5d8a-cd77-422f-a5fa-c2f3fc6a6dd0"
      ],
      "maxLatitude": 50.7917,
      "longitude": -1.3624134644062649,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-10-31T14:44:00.000+0000",
      "latitude": 56.658146752760736,
      "estimatedRestoration": "2022-11-01T19:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KC8455'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KC8455",
      "reference": "KC8455",
      "maxLongitude": -3.7624,
      "minLatitude": 56.6552,
      "name": "PH9 Area",
      "networkId": "com.sse.ssepd.shepd",
      "affectedAreas": [
        "PH15 2QY",
        "PH9 0PG",
        "PH9 0PL",
        "PH9 0PP"
      ],
      "location": {
        "coordinates": [
          [
            [
              -3.7867,
              56.6661
            ],
            [
              -3.7624,
              56.6661
            ],
            [
              -3.7624,
              56.6552
            ],
            [
              -3.7867,
              56.6552
            ],
            [
              -3.7867,
              56.6661
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-10-31T17:30:00.000+0000",
      "UUID": "2c2e5d5b-1ad8-4e72-adc5-ef5e362c4611",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -3.7867,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 56.6661,
      "longitude": -3.774334599567199,
      "resolved": false
    }
  ]
}
